src/Microsoft.AspNet.Mvc.Core/Rendering/ViewEngine/IView.cs

```diff
@@ -5,8 +5,21 @@
 
 namespace Microsoft.AspNet.Mvc.Rendering
 {
+    /// <summary>
+    /// Specifies the contract for a view.
+    /// </summary>
     public interface IView
     {
+        /// <summary>
+        /// Gets the path of the view as resolved by the <see cref="ViewResult"/>.
+        /// </summary>
+        string Path { get; }
+
+        /// <summary>
+        /// Asynchronously renders the view using the specified <paramref name="content"/>.
+        /// </summary>
+        /// <param name="context">The <see cref="ViewContext"/>.</param>
+        /// <returns>A <see cref="Task"/> rthat represents the asynchronous rendering.</returns>
```

Comment:
- typo: "that"
- avoid "represents" since it's meaningless.  suggest `A <see cref="Task"/> that on completion renders the view.`